data/talks/2018/03/crafting-quality-php-applications-an-overview.json

```diff
@@ -0,0 +1,11 @@
+{
+    "title": "Crafting Quality PHP Applications: an overview",
+    "abstract": "This prototype works, but it's not pretty, and now it's in production. That legacy application really needs some TLC. Where do we start? When creating long lived applications, it's imperative to focus on good practices. The solution is to improve the whole development life cycle; from planning, better coding and testing, to automation, peer review and more. In this talk, we'll take a quick look into each of these areas, looking at how we can make positive, actionable change in our workflow.",
+    "event": "code-quality",
```

Comment:
@DaveLiddament @opdavies Question :) Given potential event name/slug clashes (e.g. "Lightning talks"), should we go with the implicit assumption that we'll look in a matching dir for the event file? Ie. if the talk file is in 2018/03, we look for a matching event under the events/2018/03 dir?

Otherwise we'll need to somehow specify a full path to the event file, but I don't think that's necessary? I think assuming a matching <year>/<month> dir for talks and events should be fine? Just wondering if it will work for you implementation-wise, or would you prefer a different structure in the json files?